src/components/ModalBody/ModalBody.tsx

```diff
@@ -0,0 +1,24 @@
+import clsx from 'clsx';
+import React, { ReactNode } from 'react';
+import styles from './ModalBody.module.css';
+
+type Props = {
+  /**
+   * Child node(s) that can be nested inside component. `ModalHeader`, `ModalBody`, and `ModelFooter` are the only permissible children of the Modal
+   */
+  children: ReactNode;
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+};
+
+export const ModalBody = (props: Props) => {
+  return (
+    // This element is tabbable to allow keyboard users to scroll long content.
+    // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
```

Comment:
Is it though?

Reply:
It is not 😅, will remove

Reply:
But should it be? My question is actually that I think you're missing the tab index here. That could be set based on the value of `isScrollable`.

Reply:
The modal scrolls without tabindex with up and down arrow keys as long as there's a focusable element, but I guess we run into the case with no focusable elements (not even the close button if hidden). Wasn't a fan of allowing focus on the body but I will bring that back since I don't see an alternative

Reply:
>Wasn't a fan of allowing focus on the body but I will bring that back since I don't see an alternative

Yeah it's kinda annoying.. Make sure the focus outline is fully visible too; I ran into issues with that in the LP component.

Reply:
Good call, offset the outline by negative px to make it visible 
![image](https://user-images.githubusercontent.com/86632227/172435980-a721f298-0966-4065-9072-53dbfdf691fe.png)